### DISCUSSION TITLE
[Jetbrains Plugin] Stops working after too much text in a csv translation file

### DISCUSSION THREAD

**Discussion by philfontaine** [community/community#14712] [Copilot Conversations] [answered]
[bug-translations-copilot.csv](https://github.com/github/feedback/files/8464422/bug-translations-copilot.csv)

The file is very small (121 lines).

Passed a certain file size (or I don't know the exact condition), the copilot suggestions just stop working. However, the exact same file works fine in the VS Code copilot extension (and it can handle files with more than a thousand lines).

If you remove the last 30 lines, it will start working again.

**Comment by jansorg** [comment, marked as answer]
Thank you for the code sample! The next update will contain an improvement. Please let us know if the new update still is not showing suggestions for your use-cases.

**Reply by philfontaine** [reply]
Thank you for looking into this. I upgraded to the latest version 1.1.16.1219. It fixed the problem for the file I provided, but I still encountered the bug on the original translations file. I removed lines until it stopped breaking. It's now at 68 lines and it's not working. If I remove the last line (or any line), it starts working again.

[bug-translations-copilot-2.csv](https://github.com/github/feedback/files/8477213/bug-translations-copilot-2.csv)

**Reply by jansorg** [reply]
@philfontaine Thanks! Unfortunately this can't be solved in the near future. A workaround is to use the nightly builds, which use a smarter way for larger files or files with Unicode. You can activate nightly builds at `Preferences > Languages & Frameworks > GitHub Copilot > Update Channel`. Make sure to fetch an update after that, e.g. via `Help > Check for updates`.

**Reply by philfontaine** [reply]
Thanks. I tested the nightly but I got a JSON exception. I sent the exception stack trace through the plugins bug report so you should be able to see it.

In any case, this is not urgent at all as I'm using VS Code for this specific file, so no worries 👍
